### PR TITLE
CVS-83057 troubleshooting steps

### PR DESCRIPTION
**Pull request by xu-yuan1**
### Details:
 - *add a new article*
 - *apply new structure for troubleshooting guide*

### Tickets:
 - *CVS-83057*

### CODE REVIEW

**Review by kblaszczak-intel — APPROVED** [review]
looks cool

File: docs/install_guides/troubleshooting-steps.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+## Check if GPU drvier is installed
+
+[Additional configurations](configurations-header.md) are required in order to use OpenVINO on different hardware.
+
+To run inference on GPU, make sure that you have installed the correct GPU driver. To check that, see [additional configurations for GPU](configurations-for-intel-gpu.md).
+
+## Firewall/Network Issues
+
+Make sure that your firewall and network settings are working as expected. For issues caused by this reason, see <a href="openvino_docs_get_started_guide_troubleshooting_issues.html#install-for-prc">Errors with Installing via PIP for PRC Users</a> and <a href="openvino_docs_get_started_guide_troubleshooting_issues.html#proxy-issues">proxy issues with installing OpenVINO on Linux from Docker</a>.
```
Comment (helena-intel) [inline docs/install_guides/troubleshooting-steps.md:84]: You can also have proxy issues apart from the specific docker and PRC contexts, so I would rephrase this as just two examples

**Review by helena-intel — CHANGES_REQUESTED** [review]
Thanks @xu-yuan1 ! I have some comments/suggestions inline. I'm also wondering if the HDDL specific things could be moved to an HDDL specific document. These issues will not affect most users and they make other information harder to find.

File: docs/install_guides/troubleshooting-steps.md (hunk 1 above)
Comment (xu-yuan1) [reply docs/install_guides/troubleshooting-steps.md:84]: Actually the two examples are specific: the PRC one uses aliyun as a mirror repo (because PRC users constantly have this proxy issue), and the proxy example is directing users to a section about how to set proxy for Docker installation.

File: docs/install_guides/troubleshooting-steps.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+python -c "from openvino.runtime import Core"
+```
+
+If OpenVINO was successfully installed, nothing will happen. If not, an error will be displayed. 
+
+## Check if GPU drvier is installed
+
+[Additional configurations](configurations-header.md) are required in order to use OpenVINO on different hardware.
+
+To run inference on GPU, make sure that you have installed the correct GPU driver. To check that, see [additional configurations for GPU](configurations-for-intel-gpu.md).
```
[the remark this answers is not in the document]
Comment (xu-yuan1) [reply docs/install_guides/troubleshooting-steps.md:80]: Right. This will need an extra task to address it. I will create a ticket for it.

File: docs/install_guides/troubleshooting-steps.md (hunk 1 above)
Comment (helena-intel) [reply docs/install_guides/troubleshooting-steps.md:84]: Understood, but this phrasing indicates that issues caused by "firewall and network settings" are addressed in the PRC and Docker sections. People who are not in PRC and do not use Docker can also have issues with network settings (for example: "pip install" will not work if I am on VPN without configuring proxy. Usually people will probably want to set proxy settings system wide because many commands fail if you do not, but for installation issues, it might be simplest to specify options for using pip behind a proxy, like `pip install --proxy http://address:port --trusted-host pypi.org openvino`. That would allow at least installation, but a command like `omz_downloader` would still fail if generic proxy settings are not set up.

File: docs/install_guides/troubleshooting-issues.md [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+   pip install openvino-dev -i https://mirrors.aliyun.com/pypi/simple/
+   ```
+   Use the ``--trusted-host`` parameter if the URL above is ``http`` instead of ``https``.
+   You can also run the following command to install specific framework. For example:
+   
+   ```
+   pip install openvino-dev[tensorflow2] -i https://mirrors.aliyun.com/pypi/simple/
+   ```
+   
+* If you run into incompatibility issues between components after installing OpenVINO, try running ``requirements.txt`` with the following command:
```
[the remark this answers is not in the document]
Comment (xu-yuan1) [reply docs/install_guides/troubleshooting-issues.md:23]: I think it means "after installing OpenVINO Runtime, if you have compatibility issues between different components, you can solve this by using the requirements.txt to re-install the components." And this only applies to C++ developers, as using PyPI to install Runtime would not have this compatibility issue. Does it make sense?

File: docs/install_guides/troubleshooting-steps.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+```
+
+- If you don't see similar information, your PATH variable may be configured incorrectly. Check if you have written the correct <INSTALL_DIR> or tried to activate it in the right folder.
+- If you added it to a `.bashrc` file, make sure that the command is correctly written and the file is found in the `~/.bashrc` folder.
+
+## Verify if OpenVINO is correctly installed
+
+To verify if OpenVINO is correctly installed, use the following command:
+```sh
+python -c "from openvino.runtime import Core"
```
[the remark this answers is not in the document]
Comment (xu-yuan1) [reply docs/install_guides/troubleshooting-steps.md:61]: I have updated it with "For Python developers", but how can C++ developers verify whether they installed successfully?

File: docs/install_guides/troubleshooting-steps.md (hunk 1 above)
Comment (xu-yuan1) [reply docs/install_guides/troubleshooting-steps.md:73]: I see. Let me rephrase that.

File: docs/install_guides/troubleshooting-issues.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (helena-intel) [reply docs/install_guides/troubleshooting-issues.md]: Yes, thank you! Could you clarify this in the sentence? It is/was not clear what is meant with "compatibility issues between different components" means here, and how I know that the issue I'm running into is an incompatibility issue. I now understand that this addresses the situation when the OpenVINO version installed with installer/APT/etc is incompatible with the openvino-dev version. This step resolves that indeed. 

My comment was based on experience with incompatible packages after installing openvino-dev even with PyPI. That can happen when you install openvino-dev in an environment where other tools are already installed. In the installation guide, we specify to create a clean virtualenv, and this should not happen then. Nevertheless, users may also search for e.g. "openvino incompatible", find this troubleshooting entry, and assume it applies to them.

File: docs/install_guides/troubleshooting-steps.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+## Check if you have installed OpenVINO before 
+
+If you have installed OpenVINO before and added `setupvars` to your `PATH /.bashrc`, after restarting your environment, you should see the following information:
+```sh
+[setupvars.sh] OpenVINO™ environment initialized
+```
+
+- If you don't see similar information, your PATH variable may be configured incorrectly. Check if you have written the correct <INSTALL_DIR> or tried to activate it in the right folder.
+- If you added it to a `.bashrc` file, make sure that the command is correctly written and the file is found in the `~/.bashrc` folder.
```
[the remark this answers is not in the document]
Comment (xu-yuan1) [reply docs/install_guides/troubleshooting-steps.md]: Updated the content. Could you help check again?

File: docs/install_guides/troubleshooting-steps.md (hunk 1 above)
Comment (xu-yuan1) [reply docs/install_guides/troubleshooting-steps.md]: Updated. Could you help check again?

File: docs/install_guides/troubleshooting-issues.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (xu-yuan1) [reply docs/install_guides/troubleshooting-issues.md]: Updated. Please help check again. :)

File: docs/install_guides/troubleshooting-steps.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (xu-yuan1) [reply docs/install_guides/troubleshooting-steps.md:69]: https://jira.devtools.intel.com/browse/CVS-86323 was created for this.

File: docs/install_guides/troubleshooting-steps.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (xu-yuan1) [reply docs/install_guides/troubleshooting-steps.md:61]: @zKulesza Zoya, could you help provide this information? :)

File: docs/install_guides/troubleshooting-steps.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+To check your Python version, run `python -VV` or `python --version`. The supported Python versions should be 64-bit and between 3.7 and 3.9. Note that Python 3.6 is not supported anymore.
+
+If your Python version does not meet the requirements, update Python:
+
+* For Windows, **do not install Python from a Windows Store** as it can cause issues. You are highly recommended to install Python from <https://www.python.org/>.
+* For Ubuntu and other Linux systems, use the Python version comes with the system, or install the `libpython3.X` libraries via the following commands (taking Python 3.7 as an example):
+```sh
+sudo apt-get install libpython3.7
+sudo apt-get install libpython3-dev
+```
```
[the remark this answers is not in the document]
Comment (xu-yuan1) [reply docs/install_guides/troubleshooting-steps.md]: @zKulesza Hi Zoya, this one also needs your confirm. Do we need to revise it to: 
For Ubuntu and other Linux systems, use the Python version comes with the system, **and** install the `libpython3.X` libraries via the following commands (taking Python 3.7 as an example): 
```sh
sudo apt-get install libpython3.7
sudo apt-get install libpython3-dev
```

**Comment by xu-yuan1** [comment]
closing it as updates were merged to master in another PR. a new PR has created to cover further changes to troubleshooting guide: https://github.com/openvinotoolkit/openvino/pull/11896
